pkg/appgw/internaltypes.go

```diff
@@ -95,43 +107,39 @@ func getResourceKey(namespace, name string) string {
 }
 
 func generateHTTPSettingsName(serviceName string, servicePort string, backendPortNo int32, ingress string) string {
-	namePrefix := "bp-"
-	return formatPropName(fmt.Sprintf("%s%s%v-%v-%v-%s", agPrefix, namePrefix, serviceName, servicePort, backendPortNo, ingress))
+	return formatPropName(fmt.Sprintf("%s%s-%v-%v-%v-%s", agPrefix, prefixHTTPSettings, serviceName, servicePort, backendPortNo, ingress))
```

Comment:
The namespace is already in the name of the `HTTP Setting` by virtue of the `serviceName`.
Here is an example: https://github.com/Azure/application-gateway-kubernetes-ingress/blob/8b6d8fc23ff60aa9eb141503fc5bbded1e8ddf52/pkg/appgw/backendhttpsettings.go#L208
And so this could be seen in the following screenshot too (`derayche-shenzhen-ns` is the namespace): 
![image](https://user-images.githubusercontent.com/49918230/59289477-0234ae00-8c2b-11e9-8f00-c778a4f51487.png)